content/en/developers/integrations/oauth_for_integrations.md

```diff
@@ -72,19 +88,34 @@ Ensure that you do not encounter any errors when going through the basic authori
    4. Go through the OAuth flow and authorize your integration.
 
 #### Create an API Key
-If your OAuth client requests the `api_keys_write` scope, ensure that you can successfully make a request to the `marketplace_create_api` endpoint with your token in the headers of the request.
+
+If your OAuth client requests the `api_keys_write` scope, ensure that you can successfully make a request to the `marketplace` endpoint with your token in the headers of the request. 
```

Comment:
Can we link the marketplace endpoint to here: https://docs.datadoghq.com/developers/authorization/oauth2_endpoints/?tab=apikeycreationendpoints